lib/resolve-bindings-web.js

```diff
@@ -0,0 +1,9 @@
+function resolveBindings(){
+  if (navigator.bluetooth && !process.env.NOBLE_WEBSOCKET) {
+    return require('./webbluetooth/bindings');
+  }
+
+  return require('./websocket/bindings');
```

Comment:
@sandeepmistry Yup, but it's still pretty small overall when I do something like `const noble = require('noble/with-bindings')(require('noble/lib/webbluetooth/bindings'));` so no worries.